lib/resources/example_json/dstu2_examples_medication_administration.rb

```diff
@@ -0,0 +1,429 @@
+module Cerner
+  module Resources
+
+    DSTU2_MEDICATION_ADMINISTRATION_BUNDLE ||= {
+      "resourceType": "Bundle",
+      "id": "a9117d85-042f-4dda-9e38-dce0286c57a4",
+      "type": "searchset",
+      "link": [
+        {
+          "relation": "self",
+          "url": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationAdministration?patient=4642007"
+        },
+        {
+          "relation": "next",
+          "url": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationAdministration?patient=4642007&-pageContext=H4sIAAAAAAAAADWMuwoCMRBF_yW1BJN1iU4nrqVWW9hJSCYmYB6Ms4KI_25AvN3hHs5bNHtDAeJ8vMxi1Ymw8An93udUDgvXEPq7UAGHVJAgJw8uWmKJz65CiIna3b4kY25XilX-RPto0tUMRptRb4fexuInnvM_qkal9M4Mm3Xf5wtjreoSiQAAAA%3D%3D&-pageDirection=NEXT"
+        }
+      ],
+      "entry": [
+        {
+          "fullUrl": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationAdministration/7283373",
+          "resource": { # REGULAR, MLTP CODINGS
+            "resourceType": "MedicationAdministration",
+            "id": "7283373",
+            "meta": {
+              "versionId": "7283374",
+              "lastUpdated": "2017-11-22T17:25:12.000Z"
+            },
+            "text": {
+              "status": "generated",
+              "div": "<div><p><b>Medication Administration</b></p><p><b>Patient Name</b>: Smart, Connie</p><p><b>Medication Display</b>: acetaminophen</p><p><b>Was Not Given</b>: False</p><p><b>Dosage Quantity</b>: 120</p><p><b>Medication Status</b>: in-progress</p><p><b>Effective Time</b>: 2017-11-22T17:25:00.000Z</p></div>"
+            },
+            "status": "in-progress",
+            "patient": {
+              "reference": "Patient/4642007",
+              "display": "Smart, Connie"
+            },
+            "practitioner": {
+              "reference": "Practitioner/1776007",
+              "display": "Julius, Scott"
```

Comment:
Typically we want to change the returned names unless they are known to be staged data. Find and replace 'Julius, Scott' to something else. We use a lot of Doctor Who references if you poke around

Reply:
Changed the names but then realized I accidentally used one of the actors names (I'm not exactly a whovian) and found a different name: https://github.com/cerner/fhir.cerner.com/pull/169/commits/8cc029476b1b8750d7b113a67ef9c9f1f685c3dc